lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -42,6 +53,28 @@ def get_object(self, queryset=None):
 
         raise Http404
 
+    def update(self, request, *args, **kwargs):
```

Comment:
Good call; updated.  (My version of the mixin delegates more work to the superclass than the one you linked to; I think that's going to make this more maintainable in the long run).